pkg/utils/selector.go

```diff
@@ -127,6 +127,64 @@ func SelectPods(ctx context.Context, c client.Client, selector v1alpha1.Selector
 	return pods, nil
 }
 
+// CheckPodMeetSelector checks if this pod meets the selection criteria.
+// TODO: support to check fieldsSelector
+func CheckPodMeetSelector(pod v1.Pod, selector v1alpha1.SelectorSpec) (bool, error) {
+	if len(selector.Pods) > 0 {
+		for ns, names := range selector.Pods {
+			if pod.Namespace != ns {
+				continue
+			}
+
+			for _, name := range names {
+				if pod.Name == name {
+					return true, nil
```

Comment:
why return true once pod name matches? maybe other selectors(eg.`AnnotationSelectors`) don't match

Reply:
yep, you are right. I will fix it.

Reply:
oh, if the pod met this selector, it means that the pod is specified.

Reply:
This may be ambiguous, let me fix it

Reply:
we'd better let the selection continues?

Reply:
yep, you are right, I will fix it.